### PR TITLE
Update Test Fixtures

### PR DESCRIPTION
**Pull request by Integralist**
Here are some fixtures I've not updated:

- `managed_logging`: this was only generated in the last few days.
- `custom_tls`, `custom_tls_activation`, `platform_tls`, `tls`: all TLS related fixtures as they required generation of real certificates, asymmetrical keys and other similar resources.
- `kafka`: as the api backend changed behaviour and so I'm going to make that change as a separate PR so it can be explicitly called out in the dynamically updated CHANGELOG.

**NOTES**: this has been a frustrating experience, and one that has made realize we really do need to move away from fixture files for a simpler mocked/dependency injection design.

### CODE REVIEW

**Comment by Integralist** [comment]
@christophermancini as discussed on Slack, if you have capacity to review this PR please. Thank you!

File: fastly/token.go [hunk 1]
```diff
@@ -64,17 +64,17 @@ func (c *Client) ListTokens() ([]*Token, error) {
 
 // ListCustomerTokensInput is used as input to the ListCustomerTokens function.
 type ListCustomerTokensInput struct {
-	ID string
+	CustomerID string
```
Comment (phamann) [inline fastly/token.go:67]: As this is changing functionality, this feels like it should be in a separate PR and not in a test fixtures update PR. As will make documenting in the changelog easier.

File: fastly/kafka_test.go [hunk 2]
```diff
@@ -38,7 +38,7 @@ func TestClient_Kafkas(t *testing.T) {
 			TLSClientCert:    clientCert,
 			TLSClientKey:     clientKey,
 			ParseLogKeyvals:  true,
-			RequestMaxBytes:  1024,
+			RequestMaxBytes:  2048,
```
Comment (phamann) [inline fastly/kafka_test.go:41]: As this is changing functionality, this feels like it should be in a separate PR and not in a test fixtures update PR. As will make documenting in the change-log easier.

**Review by phamann — CHANGES_REQUESTED** [review]
Thanks for doing this. Changes to readme and yaml recordings look good to me. However I feel that the two functionality changes should be in separate PRs and not this one.

File: fastly/kafka_test.go (hunk 2 above)
Comment (Integralist) [reply fastly/kafka_test.go:41]: Changing the value from `1024` to `2048` was a change that _had_ to happen in order to regenerate the fixture 😬 it seems the backend endpoint changed its behaviour.

File: fastly/token.go (hunk 1 above)
Comment (Integralist) [reply fastly/token.go:67]: 👍🏻

**Comment by Integralist** [comment]
@phamann I've reverted those breaking changes, and I'll move them to a separate PR so they can appear explicitly in the dynamically generated CHANGELOG for a Go-Fastly 3.0.0 release.

> Note: this PR will also be included in that final 3.0.0 release.

/cc @christophermancini

**Review by phamann — APPROVED** [review]
👍🏻 LGTM
